django_tables2/rows.py

```diff
@@ -162,6 +165,25 @@ def _call_render(self, bound_column, value=None):
             'table': self._table,
         })
 
+    def get_cell_value(self, name):
+        '''
+        Returns the final rendered value (excluding any html) for a cell in the row, given the name
+        of a column.
+        '''
+        return self._get_and_render_with(name, self._call_value)
+
+    def _call_value(self, bound_column, value=None):
```

Comment:
I can't see a simple way of doing that without a bunch of nested/conditional `getttr`s because I've already deduped the `_get_and_render_with`